setup/README.md

```diff
@@ -1,87 +1,92 @@
-# Otto Setup Guide
+# Otto Setup Guide for Cloud Admins
```

Comment:
All references to the .env have been removed which I think is an issue

Reply:
The script prompts for the values and shows the user.